src/WealthParkOwner/app/src/main/java/com/wealthpark/owner/items/NewsItem.java

```
@@ -0,0 +1,67 @@
+package com.wealthpark.owner.items;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.datdo.mobilib.adapter.MblUniversalItem;
+import com.squareup.picasso.Picasso;
+import com.wealthpark.owner.R;
+import com.wealthpark.owner.db.models.News;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
+import butterknife.Bind;
+import butterknife.ButterKnife;
+
+/**
+ * Created by crisler on 3/3/16.
+ */
+public class NewsItem implements MblUniversalItem{
+
+    private News news;
+    private View.OnClickListener onClickListener;
+
+    @Bind(R.id.image) ImageView image;
+    @Bind(R.id.title_text) TextView titleText;
+    @Bind(R.id.date_text) TextView dateText;
+
+    public NewsItem(News news, View.OnClickListener onClickListener) {
+        this.news = news;
+        this.onClickListener = onClickListener;
+    }
+
+    @Override
+    public View create(Context context) {
+        return LayoutInflater.from(context).inflate(R.layout.item_news, null);
+    }
+
+    @Override
+    public void display(View view) {
+        ButterKnife.bind(this, view);
+        Context context = view.getContext();
+
+        // Set the event on click
+        view.setOnClickListener(onClickListener);
+
+        try {
+            int lastIndex = news.getPicture().lastIndexOf('/') + 1;
+            final String imgUrl = (news.getPicture().substring(0,lastIndex)+ URLEncoder.encode(news.getPicture().substring(lastIndex), "UTF-8"));
+
+            Picasso.with(context)
```

App will crash if `news.getPicture()` is empty. Need to catch all exceptions and show placeholder image when exception happends